source/blog/2020-06-24-support-solidus.html.markdown

```diff
@@ -0,0 +1,101 @@
+
+**Community Events:** Another awesome way to spread the word about Solidus, connect with the community, and to do your part in supporting the platform is to host a community event. We still need to put together an official guide for Solidus, but a good example of what to do is provided by Hacktoberfest. Here’s a link to their [Event Kit](https://hacktoberfest.digitalocean.com/eventkit/). Interested companies can look to them for a comprehensive list of what to do. Maybe you could commit to organizing and hosting a virtual event once a quarter. Our community would benefit in a myriad of ways from this type of engagement.
+
+Finally, if you can’t commit to any one of these three things, perhaps you could at least join us in the Solidus Stakeholders Slack chat (solidus_stakeholders) and contribute your ideas at our weekly meetings. We would seriously love to have you.
+
+The group is made up of different people who all have a significant stake in Solidus. Most members also contribute financially through Open Collective, where there are 3 tiers of support (Ambassadors, Supporters, and Enthusiasts), each contributing a different amount each month. At Karma Creative, we chose to be an Ambassador because it meant we could:
+
+- Support the core team, community, and platform we love
+- Actively shape the direction and strategy of Solidus
+- Appear in the Community section on the Solidus site
+- Get our logo out there on conference materials
+- Be recognized in the readme and mentioned during conferences
+- Have the option to join the partnership program as a Gold partner
+
+And that brings us to our next point, which is to look at how far we’ve come as a platform since Stembolt broke the news they were leaving. Because we think a huge part of this is thanks to how we function as a community. People say that you go farther together, and that couldn’t have been more true for us.
+
+Since Stembolt left, we’ve been busy. And creating a Slack channel was just the first step of many. From there, we set up organizational meetings, assigned roles, created streams of funding through recurring donations to the platform, supported and learned from each other, organized and attended conferences and, most importantly, built really solid relationships and trust within the community.
+
+And as we touched on in our talk, trust and good relationships allow us to move forward together. Without it, there’s no real way to have an honest conversation about the roadmap, next steps, collaboration, or really anything.
+
+In summary, the Support Solidus initiative was meant, first and foremost, as a *call to arms*. With your code, through becoming an Ambassador, or by donating your time, we want to remind everyone that, together, it can be done.
+
+---
+_To continue reading this article, please visit the original on the [Karma Creative](https://karmacreative.io) website: [Full Text of original Support Solidus Article](https://karmacreative.io/support-solidus)._
```

Comment:
I'm a bit concerned by the UX implications of asking people to visit another site to continue reading, although I'm on expert here and don't really have a strong opinion. @kennyadsl what are your thoughts?

Reply:
I don't have any specific concerns around continue reading on an external website but for me, the reasoning behind this is just not clear. If it's for SEO reasons, we have the canonical tag and should not impact.